source/auth/authorization.txt

```diff
@@ -32,9 +32,35 @@ simplifies management.
 The following recommendations apply to both workforce (human) and workload 
 (application) users in all :ref:`deployment paradigms <arch-center-paradigms>`.
 
+Recommendations for Authorization: Single-Region and Multi-Region
+------------------------------------------------------------------
+
+.. collapsible::
```

Comment:
question: can we remove the collapsable panels if there are no unique considerations for either deployment distribution? seems like it's not the best UX for someone to have to click down to see that there are no unique considerations for either of these

Reply:
This is how Sarah planned it. I will change collapsible to true.